authentication/auth.go

```diff
@@ -779,6 +782,8 @@ func GetDefaultValues() map[string]string {
 	defaults["PARTICIPANT_ID"] = PARTICIPANT_ID
 	defaults["FIRST_NAME"] = "John"
 	defaults["TEST_QUESTIONS"] = "F"
+	defaults["sds_dataset_id"] = "001"
```

Comment:
I think that either the default should be None because now every survey except the supplementary data one fails on launch unless you manually remove the default

Reply:
I had as mandatory when it shouldn't be so updated that now. It should only be present on launcher now if sds_dataset_id is present in the schema metadata.

Reply:
ah awesome thats an even better solution 🎉

Reply:
Was possibly failing for you as I hadn't pushed the change to make sds dataset id a uuid 😅

Reply:
The answer might be no - not so familiar with launcher but is it possible to make the dataset id default to the one from the schema metadata?

Reply:
I think the ultimate aim is to have the dataset_id generated dynamically which we will do in this ticket: https://trello.com/c/WUVvv8xO/5740-integrate-sds-with-launcher-3-sp

Reply:
Ah perfect - in that case, all looking good to me here